.gitignore

```diff
@@ -38,3 +38,4 @@ phpunit.phar
 # ignore sub directory for dev installed apps and extensions
 /apps
 /extensions
+composer.lock
```

Comment:
composer.lock should not be removed because it is needed for scrutinizer to work properly.